packages/allure-playwright/src/helpers.ts

```diff
@@ -0,0 +1,105 @@
+import { TestInfo } from "@playwright/test";
+import { Label, LabelName, LinkType } from "allure-js-commons";
+
+export const ContentTypes = {
+  label: "allure/label",
+  link: "allure/link",
+};
+
+export function label(testInfo: TestInfo, label: Label) {
+  testInfo.attachments.push({
+    name: label.name,
+    contentType: ContentTypes.label,
+    path: label.value,
```

Comment:
```suggestion
    body: label.value,
```

See https://playwright.dev/docs/api/class-testinfo#test-info-attachments